en/ui-system/components/editor/webview.md

```diff
@@ -131,7 +130,7 @@ export class WebViewCtrl extends Component {
 }
 ```
 
-#### Note: Cross-domain issues on Web platform need to be resolved by you, the developer.
+> **Note**: cross-domain issues on Web platform need to be resolved by you, the developer.
```

Comment:
cross-domain issues on Web platform need to be resolved by yourself.

Reply:
We don't actually talk like this.

What I wrote is correct, or we could also say:

cross-domain issues on Web platform need to be resolved by you, yourself.

or

Again: cross-domain issues on Web platform need to be resolved by yourself as __Cocos Creator__ cannot assist with this.

Reply:
Okay, I get it, my teacher never taught "by you, yourself" this way of writing hahaha.

Reply:
no problem! Which way would you prefer we write this?